### ISSUE TITLE
No notification for DxcDate when closing the date picker dialog

### ISSUE THREAD

**Issue by DirkBaumannDXC** [dxc-technology/halstack-react#146]
Hi,
 
I use the DxcDate and want to make a request to the server only when focus is lost (onBlur) or when user closes the date picker dialog (but not on every keystroke or on every click in the dialog).
 
When I select a date via the date picker dialog I get the "onChange", but there is no way to get notified when the dialog is closed and there is no "onBlur" called after changing the date via dialog and clicking somewhere else.
 
"onBlur" should called when the date picker dialog is closed  or there should be something like an additional "onClose" provided.
 
Best regards,
Dirk
 
Example:
 
```
() => {
  const [inputValue, changeInput] = useState("01-01-1995");
  const [isInvalid, changeIsInvalid] = useState(false);
 
  const onChange = ({ stringValue, dateValue }) => {
    changeInput(stringValue);
    changeIsInvalid(dateValue ? false : true);
    console.log("Don't want to make a request to the server on every keystroke")
  };
  const onBlur = (stringValue) => {
    changeInput(stringValue);
    console.log("Want to make a request to the server when focus lost")
  };
 
  return (
    <DxcDate
          label="Input label"
          assistiveText="assistive text"
          value={inputValue}
          invalid={isInvalid}
          placeholder
          format="MM-dd-yyyy"
          onBlur={onBlur}
          onChange={onChange}
        />
  );
}
```

**Comment by mmascord** [comment]
When will this fix be available?  It is not available in 2.0.0

**Comment by jsardid** [comment]
It's already available in @next versions (@next might bring some other breaking changes, though), but still didn't create a numbered release.
We are planing to do that soon, probably this week.

**Comment by jsardid** [comment]
Fix included in version 3.0.0
